<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Karthikas-Air" failures="4" tests="12" name="parallel.parallelrun" time="163.688" errors="1" timestamp="2024-03-28T18:11:05 PDT" skipped="0">
  <testcase classname="parallel.parallelrun" name="runScenario" time="12.596"/>
  <testcase classname="parallel.parallelrun" name="runScenario" time="15.439"/>
  <testcase classname="parallel.parallelrun" name="runScenario" time="22.709"/>
  <testcase classname="parallel.parallelrun" name="runScenario" time="12.391"/>
  <testcase classname="parallel.parallelrun" name="runScenario" time="7.113">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at parallel.Arrays_S2.message_in_console(Arrays_S2.java:137)
at ✽."Submission Successful" message displayed in console(file:///Users/karthikaramesh/eclipse-workspace/TechNinja_152/src/test/resources/parallel/Arrays_S2.feature:71)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="parallel.parallelrun" name="runScenario" time="12.394"/>
  <testcase classname="parallel.parallelrun" name="runScenario" time="12.392"/>
  <testcase classname="parallel.parallelrun" name="runScenario" time="7.076">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at parallel.Arrays_S2.message_in_console(Arrays_S2.java:137)
at ✽."Submission Successful" message displayed in console(file:///Users/karthikaramesh/eclipse-workspace/TechNinja_152/src/test/resources/parallel/Arrays_S2.feature:71)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="parallel.parallelrun" name="runScenario" time="15.411"/>
  <testcase classname="parallel.parallelrun" name="runScenario" time="15.734">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at parallel.Arrays_S2.message_in_console(Arrays_S2.java:137)
at ✽."Error occurred during submission" message displayed in console(file:///Users/karthikaramesh/eclipse-workspace/TechNinja_152/src/test/resources/parallel/Arrays_S2.feature:53)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="parallel.parallelrun" name="runScenario" time="14.669">
    <error message="element not interactable
  (Session info: chrome=123.0.6312.87)
Build info: version: &#039;4.18.1&#039;, revision: &#039;b1d3319b48&#039;
System info: os.name: &#039;Mac OS X&#039;, os.arch: &#039;aarch64&#039;, os.version: &#039;13.4&#039;, java.version: &#039;17.0.8.1&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [2a73e2dab91a8708204e7ab54ab1207a, sendKeysToElement {value=[Ljava.lang.CharSequence;@47536e4a, id=f.E25ADFD3EE3B43E2DFDA4F46A7132CCA.d.979A981C6AB5B6B74C4CBEF9E9047E19.e.44}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.87, chrome: {chromedriverVersion: 123.0.6312.86 (9b72c47a0536..., userDataDir: /var/folders/qz/lb_z2h0j7ll...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:63388}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: mac, proxy: Proxy(), se:cdp: ws://localhost:63388/devtoo..., se:cdpVersion: 123.0.6312.87, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on mac (2a73e2dab91a8708204e7ab54ab1207a)] -&gt; xpath: //form[@id=&#039;answer_form&#039;]/div/div/div/textarea]
Session ID: 2a73e2dab91a8708204e7ab54ab1207a" type="org.openqa.selenium.ElementNotInteractableException">
      <![CDATA[org.openqa.selenium.ElementNotInteractableException: element not interactable
  (Session info: chrome=123.0.6312.87)
Build info: version: '4.18.1', revision: 'b1d3319b48'
System info: os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '13.4', java.version: '17.0.8.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [2a73e2dab91a8708204e7ab54ab1207a, sendKeysToElement {value=[Ljava.lang.CharSequence;@47536e4a, id=f.E25ADFD3EE3B43E2DFDA4F46A7132CCA.d.979A981C6AB5B6B74C4CBEF9E9047E19.e.44}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.87, chrome: {chromedriverVersion: 123.0.6312.86 (9b72c47a0536..., userDataDir: /var/folders/qz/lb_z2h0j7ll...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:63388}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: mac, proxy: Proxy(), se:cdp: ws://localhost:63388/devtoo..., se:cdpVersion: 123.0.6312.87, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on mac (2a73e2dab91a8708204e7ab54ab1207a)] -> xpath: //form[@id='answer_form']/div/div/div/textarea]
Session ID: 2a73e2dab91a8708204e7ab54ab1207a
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:519)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:223)
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:110)
at com_Pages.Arrays_Page_S2.sendcode_SIA(Arrays_Page_S2.java:261)
at parallel.Arrays_S2.Type_valid_code(Arrays_S2.java:123)
at ✽.Type valid code for question in editor box Searchthearray(file:///Users/karthikaramesh/eclipse-workspace/TechNinja_152/src/test/resources/parallel/Arrays_S2.feature:69)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase classname="parallel.parallelrun" name="runScenario" time="15.764">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at parallel.Arrays_S2.message_in_console(Arrays_S2.java:137)
at ✽."Error occurred during submission" message displayed in console(file:///Users/karthikaramesh/eclipse-workspace/TechNinja_152/src/test/resources/parallel/Arrays_S2.feature:53)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- parallel.parallelrun -->
